sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBaseParser.g4

```diff
@@ -1079,15 +1077,13 @@ stringLit
     ;
 
 comment
-    : STRING
-    | {!double_quoted_identifiers}? DOUBLEQUOTED_STRING
+    : stringLit
     | NULL
     ;
 
 version
     : INTEGER_VALUE
-    | STRING
-    | {!double_quoted_identifiers}? DOUBLEQUOTED_STRING
+    | stringLit
```

Comment:
`stringLit` can just be `String`? I am not sure if this will make a conflict with existing `STRING` though.

Reply:
oh I guess the original PR was moved the double quoted identifiers/strings out of the `STRING`. Maybe it was hard to merge these together.

Please ignore my comment if this does not make sense.